ServiceWebsite/ServiceWebsite.Services/HearingsService.cs

```diff
@@ -43,7 +43,40 @@ private static bool HearingContainsParticipant(HearingDetailsResponse response,
         {
             return response.Participants.Any(p => p.Username.Equals(username, StringComparison.OrdinalIgnoreCase));
         }
-        
+
+        public async Task<Guid> GetParticipantId(string username, Guid id)
+        {
+            try
+            {
+                var hearingResponse = await _bookingsApiClient.GetHearingDetailsByIdAsync(id);
+                var participant = hearingResponse.Participants.First(p => p.Username.Equals(username, StringComparison.OrdinalIgnoreCase));
```

Comment:
You're using `.First` here which would throw an exception if the user isn't found. That means the next `if` statement to check if the participant is null would never be reached. Can you double check if you have a test case for this as well? I think if you do changing it to a `FirstOrDefault` might solve it

Reply:
Resolved